contracts/UBI.sol

```diff
@@ -80,6 +87,32 @@ contract UBI is Initializable {
 
   /// @dev Timestamp since human started accruing.
   mapping(address => uint256) public accruedSince;
+  
+  /// @dev Timestamp of PoH activation, used as internal PoH Marker
+  mapping(address => bool) public activated;
```

Comment:
IMPORTANT: Modeling it this way will require setting the boolean for everyone who is already accruing. So either need to gas a bulk update for all the existing accounts, have accounts reactivate themselves, or model this differently.

Reply:
@santisiri any thoughts on how to handle this in upgrade pattern?

WE an add a transitional code in transfer function and remove those lines after everyone is on other side.. 
+ a backup upgrade function to manually update last remaining users.